vpr/src/server/bytearray.h

```diff
@@ -16,29 +16,93 @@ class ByteArray : public std::vector<char> {
 public:
```

Comment:
It would be good to expand the comment on ByteArray from "as a simple wrapper" to also explain what its purpose is (why do we need this class?)

Reply:
solved
https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/eccfeda00ebc8e927123f1e30878e786396c7692
(wll be updated later)